src/compass/utils/lut.py

```diff
@@ -0,0 +1,29 @@
+def compute_geocoding_correction_luts(burst, xstep=500, ystep=50):
```

Comment:
I do not think it needs to be addressed in this PR (because it would need a quite of work), but we might need to have an option to provide azimuth / range starting times by adding the arguments for these. That was necessary especially when I was defining the correction grids based on ETAD product.
If the starting time(s) is not provided, then we can bring them from `burst`

Reply:
Do we need to add this in the runconfig/schema? Would it be fine if the azimuth FM rate mismatch PR would take care of this?

Reply:
> Do we need to add this in the runconfig/schema? 

Depends on how we define the correction grid on COMPASS. If we are going to define the correction grid **without** the starting timing for COMPASS production, then we wouldn't need to add them in runconfig.

Maybe adding the starting time would be more relevant to `s1-reader` than to `COMPASS`. I think adding the flexible starting times would be more relevant features when we are comparing our calculation with another source of computation (e.g. ETAD)

I will definitely include more freedom in timing when adding FM rate mismatch on `s1-reader`, but not sure if I am going to touch other corrections (e.g. `bistatic_delay`, `geometrical_and_steering_doppler`) because I have limited understanding to those.

Reply:
So, I would suggest that we hold off on that :)